docs/specs/ecma2Yaml.yml

```diff
@@ -215,7 +215,7 @@ repos:
       
 outputs:
   cat/.errors.log: |
-    {"message_severity":"warning","code":"ECMA2Yaml_ExceptionTypeNotFound","message":"Referenced exception type not found: None.Existing.Ex", "file":"*xml/CatLibrary/Cat`2.xml*"}
+    {"message_severity":"warning","code":"ECMA2Yaml_ExceptionTypeNotFound","message":"Referenced exception type not found: None.Existing.Ex", "file":"_repo.en-us/CatLibrary/Cat`2.xml"}
```

Comment:
This leads me to think we can use `_repo.en-us` as a way to detect if it is fallback error in `OpsPreProcessor.cs` line 70?

Reply:
you mean refine all the fallback errors with `_repo.en-us`?

Reply:
This require ErrorLog to load config firstly, otherwise the log will still be output.
https://github.com/dotnet/docfx/blob/v3/src/docfx/lib/log/ErrorLog.cs#L111

Reply:
yep